R/inspectMetaDifferences.R

```diff
@@ -27,13 +29,10 @@
 #'
 #'@export
 inspectMetaDifferences <- function(varName, GADSdat1, GADSdat2) {
-  check_GADSdat(GADSdat1)
-  check_GADSdat(GADSdat2)
-  if(!is.character(varName) || length(varName) != 1) stop("'varName' must be a character of length 1.")
-  if(!varName %in% namesGADS(GADSdat1)) stop("'varName' is not a variable in 'GADSdat1'.")
-  if(!varName %in% namesGADS(GADSdat2)) stop("'varName' is not a variable in 'GADSdat2'.")
-
-  #if(isTRUE(all.equal(GADSdat2$dat[, varName], GADSdat1$dat[, varName], scale = 1))) return("all.equal")
+  check_characterArgument(varName, argName = "varName")
+  check_vars_in_GADSdat
+  if(!varName %in% unlist(namesGADS(GADSdat1))) stop("'varName' is not a variable in 'GADSdat1'.")
+  if(!varName %in% unlist(namesGADS(GADSdat2))) stop("'varName' is not a variable in 'GADSdat2'.")
```

Comment:
Incorporated remarks via `check_vars_in_GADSdat()` and added tests. Agree that `cli` could be an option for later, but this would definetely be a bigger project.